indexing-service/src/main/java/io/druid/indexing/common/index/YeOldePlumberSchool.java

```diff
@@ -113,6 +114,10 @@ public Plumber findPlumber(
     // Set of spilled segments. Will be merged at the end.
     final Set<File> spilled = Sets.newHashSet();
 
+    // IndexMerger implementation.
+    final IndexMerger theIndexMerger = config.getCustomIndexMerger() != null
```

Comment:
usually the default implement is provided in the class context, like `indexMergerV9` here. Also the `TaskToolbox` has the default implement, like toolbox.getIndexMergerV9().
So I think it's better to regard the old code here to get the default implement.